controllers/storageclassrequest/storageclassrequest_controller.go

```diff
@@ -425,6 +476,41 @@ func (r *StorageClassRequestReconciler) reconcileCephBlockPool(storageProfile *v
 	return nil
 }
 
+func (r *StorageClassRequestReconciler) reconcileRadosNamespace(storageProfile *v1.StorageProfile) error {
+	_, err := ctrl.CreateOrUpdate(r.ctx, r.Client, r.cephRadosNamespace, func() error {
+		if err := r.own(r.cephRadosNamespace); err != nil {
+			return err
+		}
+
+		addLabel(r.cephRadosNamespace, controllers.StorageConsumerNameLabel, r.storageConsumer.Name)
```

Comment:
What is the reason for having this?

Reply:
I find it useful in `oc get` statements in my testing.

Reply:
Ok good enough reason, can you comment above the label so a future contributor will not remove it when they see it is not in use